src/main/java/org/mskcc/cbio/oncokb/web/rest/MailsController.java

```diff
@@ -117,4 +141,47 @@ public List<MailTypeInfo> getMailsTypes() {
         return mailTypeInfos;
     }
 
+    @GetMapping("mails/termination-warning/{id}")
+    public TerminationEmailDTO getTerminationWarningEmail(@PathVariable Long id) {
+        CompanyDTO company = this.companyService.findOne(id).orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND));
+        List<UserDTO> users = this.userService.getCompanyUsers(id);
+
+        String formattedDate;
+
+        try {
+            Instant date = company.getAdditionalInfo().getLicense().getTermination().getDate();
+            ZonedDateTime zoned = date.atZone(ZoneId.of(Constants.NY_ZONE_ID));
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMMM d, yyyy");
+            formattedDate = formatter.format(zoned);
+        } catch (NullPointerException e) {
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Please save the termination date first.");
+        }
+
+        EmailAddresses addresses = this.applicationProperties.getEmailAddresses();
+
+        String from = addresses.getTechDevAddress();
```

Comment:
This should from license address

Reply:
When I do my local testing it seems to always send from the dev email.

Reply:
Probably because you are using the gmail account? That account only has one address. In production, we will be able to switch to other email addresses.